### DISCUSSION TITLE
Missed the QR code for challenge of lesson 12

### DISCUSSION THREAD

**Discussion by tbrannt** [Cyfrin/foundry-full-course-cu#3138] [General]
Can someone tell me the timestamp where the QR code for lesson 12 is shown in this video?  https://www.youtube.com/watch?v=wUjYK5gwNZs 
I must have missed it and even when skipping through the lesson again i wasn't able to find it

**Comment by EngrPips** [comment]
I can't remember seeing those QR codes myself but you can read this discussion https://github.com/Cyfrin/foundry-full-course-cu/discussions/2018 It helps on how to get the challenge contracts.

**Reply by tbrannt** [reply]
thx!

**Reply by EngrPips** [reply]
Anytime, Friend. Keep crushing it.
